src/main/java/com/paneedah/weaponlib/compatibility/CompatibleExtraEntityFlags.java

```diff
@@ -118,8 +116,4 @@ public NBTBase serializeNBT() {
     public void deserializeNBT(NBTBase nbt) {
         capabilityContainer.getStorage().readNBT(capabilityContainer, instance, null, nbt);
     }
-
-    public static boolean isProning(EntityPlayer player) {
```

Comment:
This is like the 4th one of these exact same methods that I've removed lmao.
We have an issue with duplicate code.